tools/esp8266/webApi.cpp

```diff
@@ -364,6 +368,15 @@ bool webApi::setCtrl(DynamicJsonDocument jsonIn, JsonObject jsonOut) {
             else
                 return false;
         }
+        else if(CleanState_LockAndAlarm == cmd) {
+            Inverter<> *iv = getInverter(jsonIn, jsonOut);
```

Comment:
@lumapu can we pull the getInverter and condition NULL != iv with the return false out ?
This would make the code more concise.